coral-erm/coral
# Issue 411: Pricing formulas

See Issue #411 for description.

Test plan:
  - Apply the database update (`resources/install/protected/3.0.0/005-411.sql` (will move prior merging))
  - Create a formula in resources admin -> pricing formulas
![pricing_1](https://user-images.githubusercontent.com/1679997/50159967-5bed9880-02d8-11e9-8229-7b531de9f421.png)
  - Edit the cost history for a resource
  - Check that the formula you created is available in the dropdown list
![pricing_2](https://user-images.githubusercontent.com/1679997/50160021-79bafd80-02d8-11e9-8c71-2e21d44d0834.png)
  - Once the formula is selected, fill values for the formula, and click save
  - Check that the payment amount is populated according to the formula values.
![pricing_3](https://user-images.githubusercontent.com/1679997/50160090-aa9b3280-02d8-11e9-8b29-b2e358ea655b.png)

Once the cost informations are saved, the formula is displayed with its values on cost history edition.
